geoprocessing/test/index.html

```diff
@@ -80,54 +88,65 @@
             'minzoom': 1,
             'maxzoom': 22
         },
+        // {
```

Comment:
Commented code can be removed from source. If this piece may be usable in future, we can always link to particular SHA in codebase on wiki/gist etc.